doc/source/_rulesets/branch_naming.json

```
@@ -27,7 +27,7 @@
       "type": "branch_name_pattern",
       "parameters": {
         "operator": "regex",
-        "pattern": "^(feat|fix|chore|docs|style|refactor|test|testing|perf|ci|no-ci|build|dependabot|release|maint|junk|)\\/.*",
+        "pattern": "^(feat|fix|chore|docs|style|refactor|test|testing|perf|ci|no-ci|build|dependabot|release|maint|junk|chore|style|refactor|perf|ci|build)\\/.*",
```

I would rev
```suggestion
        "pattern": "^(feat|fix|chore|docs|style|refactor|test|testing|perf|ci|no-ci|build|dependabot|release|maint|junk|)\\/.*",
```
Already implemented them in my previous suggestion @AlejandroFernandezLuces - please verify next time

---

My bad... I added them to the contributing guide since we are enforcing them.